Source/WebCore/platform/graphics/coretext/FontCoreText.cpp

```diff
@@ -315,6 +315,13 @@ static void injectTrueTypeCoverage(int type, int selector, CTFontRef font, BitVe
     unionBitVectors(result, source.get());
 }
 
+bool Font::supportsOpenTypeAlternateHalfWidths() const
+{
+    if (m_supportsOpenTypeAlternateHalfWidths == SupportsFeature::Unknown)
+        m_supportsOpenTypeAlternateHalfWidths = supportsOpenTypeFeature(getCTFont(), CFSTR("halt")) ? SupportsFeature::Yes : SupportsFeature::No;
+    return m_supportsOpenTypeAlternateHalfWidths == SupportsFeature::Yes;
+}
```

Comment:
I guess one question would be, should we be checking if it's a listed feature or should we be testing a glyph (e.g. wide brackets) to see if it has an effect? Is it likely to have fonts that would claim to support it, but not have usable support, e.g. because they're working off of a template that injects empty gpos tables or something?